test.txt

```diff
@@ -0,0 +1 @@
+Yayyyy
```

Comment:
I'm just going to combine a few paragraphs that might be typical for a PR review.

Maybe I'll include a[ link](https://webhook.site/#!/18b7f88f-7795-4b54-8c50-e2d14c0aecc0/45e1bc13-0487-4a2e-9084-79bab973e9bd/1) as well, since that's probably pretty likely to happen in a PR review.

Mostly I'm just trying to see if the full event payload will contain a long body like this and how it will be formatted.

Reply:
I wonder if the review gets longer if I comment again. 

I'm just going to combine a few paragraphs that might be typical for a PR review.

Maybe I'll include a[ link](https://webhook.site/#!/18b7f88f-7795-4b54-8c50-e2d14c0aecc0/45e1bc13-0487-4a2e-9084-79bab973e9bd/1) as well, since that's probably pretty likely to happen in a PR review.

Mostly I'm just trying to see if the full event payload will contain a long body like this and how it will be formatted.